src/API/TaskID.h

```diff
@@ -12,7 +12,12 @@ class TaskID {
   TaskID();
 
  public:
-  unsigned int GetID() const;
+  unsigned int Get() const;
+
+  bool operator==(const TaskID& input) const;
```

Comment:
It's a good idea to separate operators overloading from class :)

Reply:
For what? :thinking: